v1.21/openstack-magnum/README.md

```diff
@@ -0,0 +1,55 @@
+# OpenStack Magnum (Kubernetes) Conformance
+
+## Create Kubernetes Cluster
+
+Setup an OpenStack environment with devstack at master branch. Then create your personal keypair by command:
+
+	openstack keypair create my-key --public-key ~/.ssh/id_rsa.pub
+
+Now you can create a Kubernetes cluster template by command:
+
+	openstack coe cluster template create k8s --network-driver calico --flavor ds4G --master-flavor ds4G --coe kubernetes --external-network public --image fedora-coreos-33.20210117.3.2-openstack.x86_64
+
+Then you can create a Kubernetes cluster by below command:
+
+	openstack coe cluster create k8s-calico-coreos --cluster-template k8s --labels=kube_tag=v1.20.2-rancher1 --node-count=2
```

Comment:
Doesn't "--labels=kube_tag=v1.20.2-rancher1" change k8s version to 1.20.2" ?

Reply:
yep. This will depend on how the images are actually cloned into the registry https://github.com/openstack/magnum/blob/c07628bca6db7afbd5f7344b1cef46213fc8ba55/magnum/drivers/common/templates/kubernetes/fragments/install-clients.sh#L9

and btw, good catch, there is a typo here. The test was executed running **v1.21.1-rancher1**